.github/ISSUE_TEMPLATE/out_of_date_package_report.md

```diff
@@ -1,45 +1,39 @@
 ---
 name: Out-of-date package reports
 about: For packages that are out-of-date
-title: ''
+title: 'Update request: PACKAGENAME OLDVERSION → NEWVERSION'
 labels: '9.needs: package (update)'
 assignees: ''
 
 ---
 
+##### Package details
```

Comment:
If we rework this template as I proposed in the other comment, we may not need any headings at all, or maybe at most for maintainers. And in that case it may as well be `##`.

The note for maintainers at the end could also be less prominent:

```
---

Maintainers: Please reference this issue in your pull request.
```